examples/line.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import vedo
+
+vedo.settings.default_backend = "vtk"
+
+from brainrender import Scene
+from brainrender.actors import Points, Line
+
+# Display the Allen Brain mouse atlas at 25 micrometer resolution.
+scene = Scene(atlas_name="allen_mouse_25um")
```

Comment:
As above, this is the default atlas, so I think the example can be simplified
```suggestion
scene = Scene()
```

Reply:
Sorry, but I disagree. [Explicit is better than implicit](https://peps.python.org/pep-0020/), and while you may know that this is the default people reading the docs are unlikely to know that (I didn't until just now!).

Reply:
Fair enough!